src/Hooks/cardContext.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const CardContext = React.createContext();
```

Comment:
Let's create default value.

Reply:
Added default value and type for context.